AntWars/GamePanel.cs

```diff
@@ -285,15 +285,9 @@ private bool checkMaxPoints()
 
         private void setPlayernameInStatistic(Config.Configuration config)
         {
-            // set playernames in statistic
-            if (config.Player1.PlayerName != "")
-            {
-                groupplayer1.Text = config.Player1.PlayerName;
-            }
-            if (config.Player2.PlayerName != "")
-            {
-                groupplayer2.Text = config.Player2.PlayerName;
-            }
+            groupplayer1.Text = "Spieler 1";
```

Comment:
Der Namen soll man eigentlich immernoch setzten können, vielleicht durch eine Methode in der AI ``String getPlayerName()``